cmd/ci-operator/main_test.go

```diff
@@ -371,3 +375,56 @@ func TestErrWroteJUnit(t *testing.T) {
 		t.Errorf(`expected full reason for error to be "something", but got %q"`, results.FullReason(defaulted))
 	}
 }
+
+func TestBuildPartialGraph(t *testing.T) {
```

Comment:
This testcase doesn't really provide a lot of value as the returned error is always the same but I needed it in order to reproduce the issue